performance_manager/lib/l2_headways.py

```diff
@@ -0,0 +1,383 @@
+from typing import Optional
+import sqlalchemy as sa
+
+from .logging_utils import ProcessLogger
+from .postgres_utils import DatabaseManager
+from .postgres_schema import (
+    VehiclePositionEvents,
+    FullTripEvents,
+    TripUpdateEvents,
+    StaticStops,
+    Headways,
+    TempHeadways,
+)
+
+
+def get_min_timestamp(db_manager: DatabaseManager) -> Optional[int]:
```

Comment:
this is another one of these functions?

Reply:
This one we actually need the `int` value being returned. Because it is used in `vp_query` and `tu_query`